edexOsgi/com.raytheon.uf.edex.plugin.grid/utility/common_static/base/menus/grid/WaveWatch.xml

```diff
@@ -31,10 +31,6 @@
 			menuText="Primary Wave Period" id="" useReferenceTime="true">
 		<substitute key="frameCount" value="${frameCount}"/>
 	</contribute>
-	<contribute xsi:type="bundleItem" file="bundles/grid/WaveWatch-SecondaryPeriod.xml"
```

Comment:
Is this commented out because we don't get this data?  (This isn't in v18 either?)